benchmark.py

```diff
@@ -226,22 +268,52 @@ def main():
   parser.add_argument("--sleep_between_retry",
                       help="number of seconds to sleep between each retry.",
                       type=int, default=5)
-
-  create_del_parser = parser.add_mutually_exclusive_group(required=False)
-  create_del_parser.add_argument("--create_delete", dest="create_delete",
-                                 action="store_true",
-                                 help="if you want to create/delete new VM.")
-  create_del_parser.add_argument("--no-create_delete", dest="create_delete",
-                                 action="store_false")
-  parser.set_defaults(create_delete=True)
-
-  skip_setup_parser = parser.add_mutually_exclusive_group(required=False)
-  skip_setup_parser.add_argument("--setup", dest="setup",
-                                 action="store_true",
-                                 help="if you want to run setup.")
-  skip_setup_parser.add_argument("--no-setup", dest="setup",
-                                 action="store_false")
-  parser.set_defaults(setup=True)
+  parser.add_argument("--db_instance_name",
+                      help="the name of the gcloud instance",
+                      default="envoy-db-instance")
+  parser.add_argument("--tier", help="the tier of GCloud SQL service",
+                      default="db-n1-standard-2")
+  parser.add_argument("--db_username", help="username on the DB",
+                      default="root")
+  parser.add_argument("--db_password",
```

Comment:
Passwords on command-line are dangerous (since anyone can view them via ps). Are there alternatives? An environment variable might make sense, not sure.